jetty-core/jetty-websocket/jetty-websocket-core-common/src/main/java/org/eclipse/jetty/websocket/core/util/WebSocketUtil.java

```diff
@@ -11,15 +11,15 @@
 // ========================================================================
 //
 
-package org.eclipse.jetty.websocket.core.internal;
+package org.eclipse.jetty.websocket.core.util;
 
 import java.nio.charset.StandardCharsets;
 import java.security.MessageDigest;
 import java.util.Base64;
 
 import org.eclipse.jetty.websocket.core.WebSocketConstants;
 
-public final class WebSocketCore
+public final class WebSocketUtil
```

Comment:
All other classes in this package are named with the plural, so rename to `WebSocketUtils`.

Reply:
We use this naming convention in other places like `BufferUtil`, `StringUtil`, `ArrayUtil`

Reply:
@lachlan-roberts singular is wrong as it is supposed to be a collection of utility methods, and we use the plural in many other cases.

Look at that package where `WebSocketUtil` is and you'll find: `InvokerUtils` (plural), `ReflectUtils` (plural), `TextUtils` (plural).
At least in the same package let's use the same naming.

Reply:
@lachlan-roberts please rename this as plural.